### PR TITLE
Fix Lesson Notification From Bug

### PR DESCRIPTION
**Pull request by michaelwehrley**
* Use the from configuration to send emails, not application name.

[#105006670]

### CODE REVIEW

**Comment by michaelwehrley** [comment]
I wasn't thinking! ARG!

File: spec/dummy/config/environments/test.rb [hunk 1]
```diff
@@ -35,7 +35,7 @@
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
   config.action_mailer.default_url_options =
-    { host: "stepped_care-no-reply@northwestern.edu" }
+    { host: "stepped_care.northwestern.edu" }
```
Comment (michaelwehrley) [inline spec/dummy/config/environments/test.rb]: @ericcf FYI, I know this has no impact on the real application, but I needed to do this so things made more sense in testing.

Comment (ericcf) [inline spec/dummy/config/environments/test.rb]: Yeah, the host probably shouldn't be an email address! I expect this was originally a case of copy and paste.

**Comment by ericcf** [comment]
:nice: :black_joker:

**Pull request merged**